pontoon/base/tests/__init__.py

```diff
@@ -100,11 +104,16 @@ class TranslationFactory(DjangoModelFactory):
     locale = SubFactory(LocaleFactory)
     string = Sequence(lambda n: 'translation {0}'.format(n))
     user = SubFactory(UserFactory)
-
     class Meta:
         model = Translation
 
 
+class IdenticalTranslationFactory(TranslationFactory):
```

Comment:
Oh, neat idea! It's never occurred to me to make useful subfactories for stuff like this.